### PR TITLE
Updated introduction.md

### PR DESCRIPTION
**Pull request by thomasantony12**
# pull request

Corrected some spelling mistakes.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/concept/elons-toy-car/.docs/introduction.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Classes
 
-The primary object-oriented construct in Java is the _class_, which is a combination of data (_fields_) and behavior (_methods_).
+The primary object-oriented construct in Java is the _class_, which is a combination of data (_fields_) and behaviour (_methods_).
```
Comment (kahgoh) [inline exercises/concept/elons-toy-car/.docs/introduction.md:5]: "behavior" is the [American spelling](https://grammarist.com/spelling/behavior-behaviour/). Exercism uses American spelling for the content (see the [Exercism style guide](https://exercism.org/docs/building/markdown/style-guide)).

Comment (thomasantony12) [reply exercises/concept/elons-toy-car/.docs/introduction.md:5]: Okay, I will change it.

**Comment by thomasantony12** [comment]
Change has been made.

**Comment by manumafe98** [comment]
Thanks for your contribution @thomasantony12 !

**Review by manumafe98 — APPROVED** [review]

**Pull request merged**